CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## 2019-11-15
```

Comment:
I don't think this is intended as it's not a release yet and should belong to the "unreleased" subsection.

Reply:
This is not resolved yet.